tslint.json

```diff
@@ -29,9 +29,9 @@
     "object-literal-sort-keys": false,
     "no-conditional-assignment": true,
     "one-line": [true, "check-open-brace", "check-whitespace"],
-    "quotemark": [true, "single"],
```

Comment:
Why do we need to change this?

Reply:
https://prettier.io/docs/en/comparison.html

and 

https://alexjover.com/blog/use-prettier-with-tslint-and-be-happy/

Please consider to use `tslint-config-prettier` because tslint often collides with prettier and then this config relaxes tslint's settings in favour of prettier.

I don't mind if you will change the settings so you will use workflow that maches your needs.

Reply:
I think it was a line that made a conflict between prettier and tslint:

https://github.com/dex4er/graphql-code-generator/blob/aab6161c668b2d18e41730934e29a0d5e13a1417/packages/plugins/typescript/document-nodes/src/index.ts#L113

Reply:
You can use template literals there instead of `"`, and I also think this prettier and tslint stuff is not directly related to this PR. So we should discuss this on another issue in my opinion.

Reply:
Agree